documentation/src/main/asciidoc/topics/con_query_index_encoding.adoc

```diff
@@ -0,0 +1,32 @@
+[id='index-encoding_{context}']
+= Cache encoding for indexing and querying
+
+To index entities in a cache, {brandname} needs a structured representation of the data.
+
+[discrete]
+== Indexing remote caches
+
+With remote caches, you add ProtoStream annotations, `@ProtoDoc("@Indexed")` and `@ProtoDoc("@Field(...)")`, to your classes to specify the fields that {brandname} indexes that describe your entities and configure indexing attributes.
```

Comment:
This is not always true, worth mentioning that you can use remote queries without any java classes or java code, only via REST and cURL, for example, or using any language that can do REST calls.